main.tf

```diff
@@ -104,6 +107,7 @@ resource "cosign_attest" "this" {
         }
         metadata = {
           invocationId = apko_build.this.id
+          startedOn    = plantimestamp()
```

Comment:
Will this not change every invocation leading to high churn?

Reply:
My ideal here would be build date epoch 🤞

Reply:
Okay, makes sense. Is there no way to include timing information in this provenance then, given existing constraints? Could we document that though this provenance lacks timestamps, the signature timestamp acts as a source of ~similar timing information for the _latest_ time the image was built; attestation signature timestamp signals the _first_ time.

Reply:
Do we need to document it?  My concern is that once we document it, it goes beyond Hyrum's Law and into the territory of documented behaviors we will ostensibly keep doing.

I think we'd probably drop the signature thing (e.g. SKIPSAME there too) if we had a better signal to use.

Reply:
The feedback I got was that timestamps were part of what good looked like, so that's what I was trying to add. But I'm fine reassessing that outside this thread. It's nice to know it's possible this way at least, if we decide this is the route we want to take.